docs/source/main/install.rst

```diff
@@ -0,0 +1,171 @@
+  pip install "apache-airflow==${AIRFLOW_VERSION}" --constraint "${CONSTRAINT_URL}"
+  pip install orloge pulp
+  pip install -r airflow_config/requirements.txt
+  pip install -e ../libs/client
+  pip install apache-airflow-providers-google
+  airflow db init
+  airflow users create -u admin -p admin -f admin -l user -r Admin -e admin@airflow.org
+  
+Then we export the variables that we need to run airflow::
+
+  export AIRFLOW__CORE__LOAD_EXAMPLES=0
+  export AIRFLOW__CORE__DAGS_ARE_PAUSED_AT_CREATION=0
+  export AIRFLOW__API__AUTH_BACKEND=airflow.api.auth.backend.basic_auth
+  export AIRFLOW__WEBSERVER__SECRET_KEY=e9adafa751fd35adfc1fdd3285019be15eea0758f76e38e1e37a1154fb36
+  export AIRFLOW_CONN_CF_URI=http://service_user:Service_password1@localhost:5000
+  airflow webserver -p 8080 &
+  airflow scheduler &
+
+And to stop the services we can::
+
+  ps aux | grep airflow
+  kill -9 <process_id>
+
+Run cornflow
+^^^^^^^^^^^^^^^^
+
+Once airflow is running and cornflow set up we can run it::
+  
+    source cenv/bin/activate
+    flask register_deployed_dags -r http://127.0.0.1:8080 -u admin -p admin
+    flask register_dag_permissions -o 1
+    flask run -p 5000 &
+
+And with this we should have both cornflow and airflow running. We can check that cornflow is running by going to http://localhost:5000/health/ and airflow by going to http://localhost:8080.
+
+
+Docker. Build images from local
+--------------------------------
+
+More information about how to deploy with docker containers can be found of the :ref:`deploy-cornflow` section of the documentation.
+
+We first start by cloning the repository::
+
+  git clone https://github.com/baobabsoluciones/cornflow
+
+Then we are going to need some files to have, that can be modified to alter how the deployment is going to work.
+
+The Dockerfile for cornflow can be found inside the ``cornflow/cornflow-server`` folder and can be used as is, as this is the Dockerfile used to build the original image on docker hub.
+
+The Dockerfile for airflow can be found on the ``cornflow/cornflow-server/airflow_config`` folder and can be used as is, as this is the Dockerfile used to build the original image on docker hub. This image is built on top of another one that gets built manually and that has all the needed libraries for the solvers. This is done this way to improve the build time of the image as the solver libraries are quite heavy and quite prone to remain stable for long periods of time.
+
+Then we need a docker-compose file. The ``docker-compose.yml`` or ``docker-compose-cornflow-celery.yml`` that are in the root folder of the repository can be used with some small tweaks to build the images from the local repo instead that taking the official images.
+
+The changes needed to be done are just to comment lines 33-34 and 63-64 and uncomment lines 35-36 and 65-66. Then we can run the following commands to start up the containers::
```

Comment:
instead of putting the line numbers, I recommend including the text to change in this file. That way if the lines change, the reference still works. Also, it's not clear if the file to change is `docker-compose.yml` or `docker-compose-cornflow-celery.yml` or if it matters.